tests/dragonfly/cluster_test.py

```diff
@@ -1122,6 +1143,11 @@ async def list_counter(key, client: aioredis.RedisCluster):
     for counter in counters:
         counter.cancel()
 
+    # clean migrations
+    for node in nodes:
+        node.migrations = []
+
+    # TODO this config should be pushed with new slots
```

Comment:
Why is this in TODO and not implemented?
if you push the prev slot configuration what are we checking in the capture? we dont actually check the data was migrated right?

Reply:
we have discussed it previous time that we have a bug with data lost in the finalization process